EU4toV2/Source/V2World/Province/Province.cpp

```diff
@@ -36,6 +36,10 @@ V2::Province::Province(std::string _filename,
 	{
 		details = mappers::ProvinceDetailsMapper("blankMod/output/history/provinces/" + filename).getProvinceDetails();
 	}
+	else if (theConfiguration.isHpmEnabled())
+	{
+		details = mappers::ProvinceDetailsMapper(theConfiguration.getVic2Path() + "/HPM/history/provinces/" + filename).getProvinceDetails();	
```

Comment:
Hasn't Vic2Path been updated to add /HPM/history? Wouldn't that make this redundant?